### PR TITLE
New employees & only showing subset each day [#121516763]

### PR DESCRIPTION
**Pull request by nc-lynchj**

### CODE REVIEW

File: src/scripts/slides.js [hunk 1]
```diff
@@ -5,95 +5,1311 @@ const people = {
...
+        'Even though Anurag already has the superpower of being the super sniffer, he said that if he had an unrealistic superpower it would be mind reading in order to understand the female mind. That\'s kind of sweet if you think about it. If he could come back to life as an animal, Anurag would want to become a dog in order to understand the dog\'s perspective on the human/dog and man\'s best friend relationship. If Anurag could be the main character in a fantasy book and had a mythological creature for his sidekick he would have a cyclops on his side to creep people out.'
+      ]
```
Comment (nc-babelv) [inline src/scripts/slides.js]: Do these bio arrays have to have 4 strings?

Comment (nc-lynchj) [inline src/scripts/slides.js]: They can be any number of strings but the paragraphs are separated for styling later.

File: src/scripts/slideshow.jsx [hunk 2]
```diff
@@ -5,17 +5,28 @@ const slides = require('./slides.js');
...
 const DISPLAY_TIME = 90000;
+const DISPLAY_RANGE = 6;
 
 const SlideShow = React.createClass({
   getInitialState() {
     return {
-      slideIndex: 0
+      slideIndex: 0,
+      minSlide: 0,
+      day: new Date().getDate(),
```
Comment (nc-babelv) [inline src/scripts/slideshow.jsx]: no comma

File: src/scripts/slideshow.jsx [hunk 3]
```diff
@@ -5,17 +5,28 @@ const slides = require('./slides.js');
...
     let nextIndex = this.state.slideIndex + 1;
-    if (nextIndex >= slides.length) {
-      nextIndex = 0;
+    if (nextIndex >= slides.length || nextIndex >= this.state.minSlide + DISPLAY_RANGE) {
+      nextIndex = this.state.minSlide;
+    }
+    const day = new Date().getDate();
+    if(day !== this.state.day) {
+      this.state.day = day;
+      if(this.state.minSlide + DISPLAY_RANGE >= slides.length) {
```
Comment (nc-babelv) [inline src/scripts/slideshow.jsx]: space after if

File: src/scripts/slideshow.jsx [hunk 4]
```diff
@@ -5,17 +5,28 @@ const slides = require('./slides.js');
...
   },
   advance() {
     let nextIndex = this.state.slideIndex + 1;
-    if (nextIndex >= slides.length) {
-      nextIndex = 0;
+    if (nextIndex >= slides.length || nextIndex >= this.state.minSlide + DISPLAY_RANGE) {
+      nextIndex = this.state.minSlide;
+    }
+    const day = new Date().getDate();
+    if(day !== this.state.day) {
```
Comment (nc-babelv) [inline src/scripts/slideshow.jsx]: space after if

File: src/scripts/slides.js [hunk 5]
```diff
@@ -5,95 +5,1728 @@ const people = {
...
   },
-  "Jonathan Pierce" : {
-    image: 'images/jon_pierce.jpg',
+  'Jonathan Pierce' : {
+    image: 'images/jonathan_pierce.jpg',
     name: 'Jonathan Pierce',
     role: 'Software Developer',
     area: 'Back'
+  },
+  'Anurag Choudhary' : {
```
Comment (nc-babelv) [inline src/scripts/slides.js]: pretty big NP, but eliminate the space between the key and the :, for this one and all the other "people"

File: src/scripts/slides.js [hunk 6]
```diff
@@ -5,95 +5,1728 @@ const people = {
...
+    image: '',
+    name: 'Jack Belcaster',
+    role: 'Ops',
+    area: ''
   }
 };
 module.exports = [
   {
     slideType: 'bio',
     args : {
```
Comment (nc-babelv) [inline src/scripts/slides.js]: same here no space after the ```args``` key

**Comment by nc-babelv** [comment]
here comes the :boom:

**Pull request merged**